.travis.yml

```diff
@@ -1,10 +1,6 @@
 sudo: false
 language: go
 go:
-  - 1.5.x
-  - 1.6.x
-  - 1.7.x
-  - 1.8.x
```

Comment:
I think it's better to keep 1.6 and above since Macaron itself supports: https://github.com/go-macaron/macaron

Reply:
thanks @Unknwon I've made update 😄